test/util/authenticate-user.js

```diff
@@ -0,0 +1,97 @@
+// Allow main functionality to stay at top of file:
+/* eslint-disable no-use-before-define */
+
+const makeFetchCookie = require('fetch-cookie');
+
+module.exports = async (service, user, includeCsrf) => {
+  if (!user) throw new Error('Did you forget the **service** arg?');
```

Comment:
When refactoring tests, I found it easy to mistakenly write `authenticateUser('alice')` instead of `authenticateUser(service, 'alice')`.  This check was very helpful in catching those mistakes - without the check here, the eventual error was nebulous.  I think it will be helpful for future test writers to keep this check here.  But this branch is passing now, so I can delete it if it's not helpful.

Alternatively, maybe the function could be refactored to be an instance method of `service`.  Would that be preferred?

Reply:
> the function could be refactored to be an instance method of service

I like the idea